Dockerfile

```diff
@@ -20,18 +19,11 @@ RUN corepack enable
 RUN yarn set version 4.1.0
 
 # This will install dependencies for the sync-daemon, devDependencies included:
```

Comment:
Is this comment still valid?

Reply:
No, updated in 3e1a48f162c8755e6042000f17493567e414a534

Thanks